packages/rum/src/domain/record/elementsScrollPositions.ts

```diff
@@ -24,3 +31,21 @@ export function createElementsScrollPositions() {
     },
   }
 }
+
+function tryToFindScrollingElement(scrollPositions: ScrollPositions) {
+  if (document.scrollingElement) {
+    return document.scrollingElement
+  }
+  if (scrollPositions.scrollLeft === 0 && scrollPositions.scrollTop === 0) {
+    addTelemetryDebug('Unable to find scrolling element for scroll (0,0)')
+    return null
+  }
+  if (
+    Math.round(document.documentElement.scrollLeft) === scrollPositions.scrollLeft &&
+    Math.round(document.documentElement.scrollTop) === scrollPositions.scrollTop
+  ) {
+    return document.documentElement
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** Previously, we used `document.scrollingElement` before any scroll occured. Now that we are using it only when a scroll occurs, ‏I'd be curious of how often we get in this case now. Would you mind adding a telemetry log here?

Reply:
is [0c65a8d](https://github.com/DataDog/browser-sdk/pull/1688/commits/0c65a8d5f03efe11f9e62a3994cdeb82f641c2e6) what you had in mind?

Reply:
Yes it does the job. The idea was to have a log at every codepath when `scrollingElement` was missing. Adding it here works for me.